e2e/npm.spec.js

```diff
@@ -0,0 +1,14 @@
+import { test, Expect } from '../fixtures.js';
+
+test.describe('npm', () => {
+  test('check for overlay and tooltip', async ({ page }) => {
+    await page.goto('https://stackoverflow.com/questions/9023672/how-do-i-resolve-cannot-find-module-error-using-node-js', {
+      waitUntil: 'domcontentloaded',
+    });
+    const moduleNameLink = page.locator('overlay-indicator[overlay-indicator-package-name="module_name"]');
+    await moduleNameLink.scrollIntoViewIfNeeded();
+    await moduleNameLink.hover();
+    const overlayModuleName = page.locator('.overlay-tooltip__tooltip .overlay-indicator__tooltip[data-testid="module_name"]');
+    await Expect(overlayModuleName).toBeVisible();
```

Comment:
It is failed in my environment, I need to understand how to debug it.

Reply:
I sent you a message in Discord

Reply:
OK it didn't work because I didn't built the extension, and it used an old version.
Now with the updated version I found two fixes.

Do you think we need to add the `build` step to the `e2e` command?

I think so. I think both of us failed in the same situation, it may save other developers' time.